modules/c/examples/code_snippets/main.c

```diff
@@ -812,6 +812,44 @@ static void group_by() {
     // end::query-groupby[]
 }
 
+static void docsonly_N1QL_Params() {
+    /* Documentation Only query-syntax-n1ql-params
+        Declared elsewhere
+          CBLDatabase* database = kDatabase;
+    */
+
+    // tag::query-syntax-n1ql-params[]
+    int errorPos;
+
+    FLSTR n1qlstr = "SELECT * FROM _ WHERE type = $type";
```

Comment:
`FLSTR` is not a type, and a slice cannot be directly set from a `const char *` anyway.